sdk-base/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentAccuracyRadiusSettings.kt

```diff
@@ -0,0 +1,22 @@
+package com.mapbox.maps.plugin.locationcomponent
+
+import android.graphics.Color
+/**
+ * Settings for accuracy radius.
+ */
+data class LocationComponentAccuracyRadiusSettings @JvmOverloads constructor(
```

Comment:
do we want to keep this class as data classes until we revisit usage of data classes in the next iteration?

Reply:
Yes, I'd like to keep as others just in case introduce other bugs.